usr/share/rear/conf/examples/blockclone.conf

```diff
@@ -0,0 +1,82 @@
+# Example blockclone.conf file used to make an image of the (USB) disk of a Raspberry Pi (RPi)
+#
+# The OUTPUT must be defined - PXE is currently the only one working with RPi
+OUTPUT=PXE
+# However, when you do not care about actually using PXE to restore define the OUTPUT_URL=null
+OUTPUT_URL=null
+# The location to store the disk image of our RPi - in this example on our local NAS storage
+BACKUP_URL=nfs://nas/volume1/RearSpace
+
+BACKUP=BLOCKCLONE                                        # Define BLOCKCLONE as backup method
+BACKUP_PROG_ARCHIVE="$(hostname -s)"                     # Name of image file (usually the hostname)
+BACKUP_PROG_SUFFIX=".dd.img"                             # Suffix of image file (dd.img indicates we use 'dd')
+BACKUP_PROG_COMPRESS_SUFFIX=""                           # Don't use additional suffixes (keep this one EMPTY)
+
+BLOCKCLONE_PROG="dd"                                     # Use dd for image creation
+BLOCKCLONE_PROG_OPTS="bs=4k"                             # Additional options that will be passed to dd
+BLOCKCLONE_PROG_COMPRESS="gzip"                          # Compress the image (optional - use gzip, ...)
+BLOCKCLONE_PROG_COMPRESS_OPTIONS="-9"                    # Options needed by the compress program (default empty)
+BLOCKCLONE_SOURCE_DEV="/dev/sda"                         # Device that should be backed up
+
+# We will add the compress suffix to the BACKUP_PROG_SUFFIX variable when
+# variable BLOCKCLONE_PROG_COMPRESS was defined
+case $BLOCKCLONE_PROG_COMPRESS in
```

Comment:
I'd move this logic to ReaR code to simplify the example configuration and to make the behavior more consistent. We should then also fail on unknown compression progs and ensure that they are part of the rescue system (for the non-RPI scenario).